osquery/database/plugins/sqlite.cpp

```diff
@@ -256,6 +261,31 @@ Status SQLiteDatabasePlugin::remove(const std::string& domain,
   return Status(0);
 }
 
+Status SQLiteDatabasePlugin::removeRange(const std::string& domain,
+                                         const std::string& low,
+                                         const std::string& high) {
+  if (read_only_) {
+    return Status(0, "Database in readonly mode");
+  }
+
+  sqlite3_stmt* stmt = nullptr;
+  std::string q = "delete from " + domain + " where key >= ?1 and key <= ?2;";
```

Comment:
Same q here, looks like things will be fine with `domain = ""`, but is there any concern about logging or notifying the user?